doc/cygnus-common/installation_and_administration_guide/running_as_service.md

```diff
@@ -0,0 +1,24 @@
+#Running Cygnus as a service
+**Note**: Cygnus can only be run as a service if you installed it through the RPM.
+
+Once the `cygnus_instance_<id>.conf` and `agent_<id>.conf` files are properly configured, just use the `service` command to start, restart, stop or get the status (as a sudoer):
```

Comment:
Running cygnus-common is the same than running a plain Flume... this sentence could be used somewhere in this paragrah :)